test/powershell/Common/TestHostCS.psm1

```diff
@@ -290,7 +304,7 @@ function New-TestHost
     }
 
     if ( ! ("TestHost.TestHost" -as "type" )) {
-       $t = add-Type -pass $definition -ref $references
+       $t = add-Type -pass $definition -ref $references -WarningAction SilentlyContinue 

```

Comment:
What warnings are being suppressed?  I assume it's a C# warning - maybe disable with a [pragma](https://msdn.microsoft.com/en-us/library/441722ys.aspx) instead so we don't ignore future warnings that maybe shouldn't be ignored.